contracts/src/v0.8/functions/dev/1_0_0/FunctionsCoordinator.sol

```diff
@@ -20,7 +20,10 @@ contract FunctionsCoordinator is OCR2Base, IFunctionsCoordinator, FunctionsBilli
     uint64 subscriptionId,
     address subscriptionOwner,
     bytes data,
-    uint16 dataVersion
+    uint16 dataVersion,
+    bytes32 flags,
+    uint64 callbackGasLimit,
+    bytes onchainMetadata
```

Comment:
Can we just build the commitment in the core node? It would save us a little chunk of gas

Reply:
Out of scope of this PR, I'll remove the TODOs for now.